utility/Adafruit_SI1145.cpp

```diff
@@ -17,9 +17,19 @@
 
 /*
  * Modified by Ben Peters (Ardusat) to avoid namespace collisions 
+ *
+ * Modified by Sam Olds (Ardusat) to also support SI1132 sensor
+ * https://hackaday.io/project/5684/logs
+ * 
+ * Modified by Sam Olds (Ardusat) to use common i2c read/write utility
  */
 
 #include "Adafruit_SI1145.h"
+#include "common_utils.h"
+
+#define read8(reg, val) readFromRegAddr(_addr, reg, val, 1, BIG_ENDIAN);
```

Comment:
I would suggest leaving out the trailing `;` so that this macro function behaves more like a "real" function:

```cpp
#define read8(reg, val) readFromRegAddr(...)

read8(REGISTER, &val);
```